dash-pipeline/bmv2/dash_outbound.p4

```diff
@@ -24,8 +24,30 @@ control outbound(inout headers_t hdr,
         }
     }
 
-    action route_vnet(bit<24> dest_vnet_vni) {
+    /* TODO:
+       If needed we can have a separate route_vnet_direct action for the
+       overlay_nh_ip override. But for this we need to fix the sai_api_gen tool
+       limitation of unique action param names across different actions in the
+       same table. Since the dest_vnet_vni param will be common for route_vnet
+       and route_vnet_direct actions.
+    */
+    action route_vnet(bit<24> dest_vnet_vni,
```

Comment:
Need to split into a separate action vnet_direct

Reply:
Done pls check.